src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/project.json

```diff
@@ -1,5 +1,6 @@
-﻿{
+{
   "dependencies": {
+    "Microsoft.VisualStudio.Threading": "15.0.240",
```

Comment:
This dependency should be picked up transitively from `NuGet.VisualStudio.Common`.